http-core/src/main/java/org/apache/pekko/http/javadsl/model/StatusCodes.java

```diff
@@ -66,10 +66,16 @@ private StatusCodes() {}
     public static final StatusCode GONE = org.apache.pekko.http.scaladsl.model.StatusCodes.Gone();
     public static final StatusCode LENGTH_REQUIRED = org.apache.pekko.http.scaladsl.model.StatusCodes.LengthRequired();
     public static final StatusCode PRECONDITION_FAILED = org.apache.pekko.http.scaladsl.model.StatusCodes.PreconditionFailed();
+    public static final StatusCode CONTENT_TOO_LARGE = org.apache.pekko.http.scaladsl.model.StatusCodes.ContentTooLarge();
+
+    /**
+     * @deprecated deprecated in favor of CONTENT_TOO_LARGE
```

Comment:
Could you include `since = "1.0.0"` on all new deprecations?

Reply:
not sure how to do this in the java comment. just add " since 1.0.0" to the comment?

Reply:
probably best to read https://www.baeldung.com/javadoc-version-since

Reply:
https://www.scala-lang.org/api/2.13.10/scala/deprecated.html describes how to set the 'since value' in Scala deprecations